src/labels/CustomLabels.labels

```diff
@@ -3130,6 +3130,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Error message for long deployment</shortDescription>
         <value>Modifying this Object Group is taking longer than expected.</value>
     </labels>
+    <labels>
+        <fullName>bdiMigrationLongDeployment</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Error message for long deployment</shortDescription>
+        <value>Converting Help Text mappings to Advanced Mapping is taking longer than expected.</value>
+    </labels>
```

Comment:
@melissabarber - just tagging you in the newly added label.

Reply:
![Screen Shot 2019-11-12 at 12 39 40 PM](https://user-images.githubusercontent.com/25848103/68708938-86c48900-0549-11ea-972b-11304ceaa19f.png)